src/drivers/dw/dma.c

```diff
@@ -334,12 +334,6 @@ static int dw_dma_start(struct dma_chan_data *channel)
 static int dw_dma_release(struct dma_chan_data *channel)
 {
 	struct dw_dma_chan_data *dw_chan = dma_chan_get_data(channel);
-	struct dma_cb_data next = {
```

Comment:
Ok, commit message describes what is happening to the pointer, can you also add why this is a bug in this situation.